src/EditorFeatures/Test/CodeFixes/CodeFixServiceTests.cs

```diff
@@ -498,7 +498,6 @@ public async Task TestNuGetAndVsixCodeFixersWithMultipleFixableDiagnosticIdsAsyn
             return await fixService.GetFixesAsync(document, TextSpan.FromBounds(0, 0), includeConfigurationFixes: false, cancellationToken: CancellationToken.None);
         }
 
-        [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
```

Comment:
❔ Why is this getting removed? I would have expected this:

```csharp
[ExportCodeFixProvider(LanguageNames.CSharp), Shared, PartNotDiscoverable]
```

Reply:
That's bizarre.  i don't remember making that change...

Reply:
Looks like it was pulled in from a separate change